modules/processing/snowflake/doc/ST_VORONOILINES.md

```diff
@@ -0,0 +1,28 @@
+### ST_VORONOILINES
+
+{{% bannerNote type="code" %}}
+processing.ST_VORONOILINES(points [, bbox])
+{{%/ bannerNote %}}
+
+**Description**
+
+Calculates the Voronoi diagram of the points provided. An array of linestrings in GeoJSON format is returned. https://turfjs.org/docs/#voronoi
+
+* `points`: `ARRAY` array of points in GeoJSON format casted to STRING.
+* `bbox` (optional): `ARRAY` clipping bounding box. By default the [-180,-85,180,-85] bbox will be used.
+
+**Return type**
+
+`ARRAY`
+
+**Example**
+
+``` sql
+SELECT sfcarto.processing.ST_VORONOILINES(ARRAY_CONSTRUCT(ST_ASGEOJSON(ST_POINT(-75.833, 39.284))::STRING, ST_ASGEOJSON(ST_POINT(-75.6, 39.984))::STRING, ST_ASGEOJSON(ST_POINT(-75.221, 39.125))::STRING, ST_ASGEOJSON(ST_POINT(-75.521, 39.325))::STRING), ARRAY_CONSTRUCT(-76.0, 35.0, -70.0, 45.0));
+-- [[ "{\"type\":\"LineString\",\"coordinates\":[[-76,39.728365000000004],[-75.72047348298037,39.63532260219203],[-75.6178875502008,38.854668674698786],[-76,37.38389622641511],[-76,39.728365000000004]]}", "{\"type\":\"LineString\",\"coordinates\":[[-70,...
+```
+
+``` sql
+SELECT sfcarto.processing.ST_VORONOILINES(ARRAY_CONSTRUCT(ST_ASGEOJSON(ST_POINT(-75.833, 39.284))::STRING, ST_ASGEOJSON(ST_POINT(-75.6, 39.984))::STRING, ST_ASGEOJSON(ST_POINT(-75.221, 39.125))::STRING, ST_ASGEOJSON(ST_POINT(-75.521, 39.325))::STRING));
```

Comment:
I thought you would use the optional parameter on this second example

Reply:
The optional bbox is being used in the first example. I can reorder the examples.